src/pages/Inventory.page.js

```diff
@@ -30,4 +30,23 @@ export class InventoryPage extends BaseSwagLabPage {
         );
         return prices;
     }
+
+    async addRandomProductsToCart(count) {
+        // Get all product elements
+        const products = await this.page.$$('.inventory_item');
+
+        // Add random products to the cart
+        for (let i = 0; i < count; i++) {
+            const randomIndex = Math.floor(Math.random() * products.length);
```

Comment:
I don't think such implementation will make test to be stable.
Have you watched the last Q&A session record, we've discussed it there.
The thing is, it will produce duplicates and you will try to click on it twice or even more times